Sources/Sentry/SentryTracer.m

```diff
@@ -29,13 +35,13 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
                            waitForChildren:(BOOL)waitForChildren
 {
     if ([super init]) {
-        _rootSpan = [[SentrySpan alloc] initWithTracer:self context:transactionContext];
+        self.rootSpan = [[SentrySpan alloc] initWithTracer:self context:transactionContext];
         self.name = transactionContext.name;
-        _children = [[NSMutableArray alloc] init];
-        _hub = hub;
-        _isFinished = YES;
+        self.children = [[NSMutableArray alloc] init];
+        self.hub = hub;
+        self.isWaitingChildren = YES;
```

Comment:
I changed the name of this variable because the class already has a property isFinished, this could cause some confusion because these two variables are used for different purpose.